src/app/services/gstorage/g_storage.py

```diff
@@ -0,0 +1,98 @@
+from google.cloud import storage
+
+class GStorage():
+
+    def __init__(self, serviceAccountPath):
+        self.service_account_path = serviceAccountPath
+    #exception = pyqtSignal(Exception)
+    
+
+    def create_bucket(self, bucket_name):
+        storage_client = storage.Client.from_service_account_json(self.service_account_path)
+        bucket = storage_client.create_bucket(bucket_name)
+        print('Bucket {} created'.format(bucket.name))
+
+    def delete_bucket(self, bucket_name):
+        storage_client = storage.Client.from_service_account_json(self.service_account_path)
+        bucket = storage_client.get_bucket(bucket_name)
+        bucket.delete()
+        print('Bucket {} deleted'.format(bucket.name))
+
+    def list_blobs(self, bucket_name):
+        """Lists all the blobs in the bucket."""
```

Comment:
Triple quote should be ''' blabla ''' instead of  """ blabla """. Triple quote create a documentation link for the function. Is it needed here?